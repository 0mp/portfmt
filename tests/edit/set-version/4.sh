${PORTEDIT} set-version 1.0.0 4.in | diff -L 4.expected -L 4.actual -u 4.expected -
