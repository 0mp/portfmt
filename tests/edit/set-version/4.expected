DISTVERSION=	1.0.0
PORTREVISION=	0
